functions.py

```diff
@@ -1,36 +1,35 @@
 import time
+
+def ReadLines(nameoffile):
+    with open(nameoffile,'r') as f:
+        arrayline = f.readlines()
+        return arrayline
+
 #how much is not taxed
 def SellWithoutTax():
-    f = open('new_register.txt','r')
-    try:
-        arrayline = f.readlines()
-        sum = 0.0
-        i = 0
+    arrayline = ReadLines('new_register.txt')
+    sum = 0.0
+    i = 0
+    arr = arrayline[i].split(',', 2)
+    while (time.time() - float(arr[1])) >= 31536000.0:
+        sum += float(arr[0])
+        i += 1
         arr = arrayline[i].split(',', 2)
-        while (time.time() - float(arr[1])) >= 31536000.0:
-            sum += float(arr[0])
-            i += 1
-            arr = arrayline[i].split(',', 2)
-        print("can sell without tax:", sum)
-        return sum, i
-    finally:
-        f.close()
+    print("can sell without tax:", sum)
+    return sum, i
 
 #benefit from not taxed
 def Benefit(sum, firsttax,price):

```

Comment:
space after firsttax, . Please install python extension in VC that formats code automatically on each save.

I couldn't understand the firsttax parameter meaning. Please come up with a more explicit name or make a doc for parameters and output.